pkg/driver/node.go

```diff
@@ -226,26 +311,25 @@ func (d *driver) NodeGetInfo(ctx context.Context, _ *csi.NodeGetInfoRequest) (*c
 }
 
 func (d *driver) NodeGetCapabilities(_ context.Context, _ *csi.NodeGetCapabilitiesRequest) (*csi.NodeGetCapabilitiesResponse, error) {
-	return &csi.NodeGetCapabilitiesResponse{
-		Capabilities: []*csi.NodeServiceCapability{
-			{
-				Type: &csi.NodeServiceCapability_Rpc{
-					Rpc: &csi.NodeServiceCapability_RPC{
-						Type: csi.NodeServiceCapability_RPC_UNKNOWN,
```

Comment:
This capability `NodeServiceCapability_RPC_UNKNOWN` has been removed. Is that intended or do we really don't need it.

Reply:
We removed the `NodeServiceCapability_RPC_UNKNOWN` capability as our CSI driver only supports the RPC methods it explicitly requires and doesn't need to indicate support for any other RPC methods.